src/main/java/com/jcabi/github/Repos.java

```diff
@@ -98,4 +106,243 @@ void remove(@NotNull(message = "coordinates can't be NULL")
     Iterable<Repo> iterate(
         @NotNull(message = "identifier can't be NULL") String identifier
     );
+
+    /**
+     * Settings to use when creating a new GitHub repository.
+     *
+     * @author Chris Rebert (github@rebertia.com)
+     * @version $Id$
+     * @since 0.24
+     * @see <a href="https://developer.github.com/v3/repos/#create">Create Repo API</a>
+     * @todo #1095:30m Add the ability to set the other parameters of
+     *  the repo creation API (has_issues, has_wiki, has_downloads,
+     *  team_id, gitignore_template, license_template).
+     */
+    @SuppressWarnings("PMD.TooManyMethods")
+    @ToString
+    @Loggable(Loggable.DEBUG)
+    @EqualsAndHashCode(of = { "nam", "priv", "descr", "home", "init" })
+    final class NewRepoSettings implements JsonReadable {
```

Comment:
@cvrebert I'd rather have it named `RepoCreate`. In the future other actions encoded in json could be `RepoDelete`, etc